pkg/KubeArmorOperator/Dockerfile

```diff
@@ -58,8 +59,24 @@ RUN groupadd --gid 1000 default \
 ARG OPERATOR_DIR=pkg/KubeArmorOperator
 COPY --from=builder --chown=1000:1000 /KubeArmor/$OPERATOR_DIR/operator /operator
 COPY LICENSE /licenses/license.txt
-RUN setcap "cap_sys_admin=+ep cap_ipc_lock=+ep cap_sys_resource=+ep cap_dac_override=+ep cap_dac_read_search=+ep" /operator
   
 USER 1000
 
-ENTRYPOINT ["/operator"]
\ No newline at end of file
+ENTRYPOINT ["/operator"]
+
+FROM redhat/ubi9-minimal as snitch
+
+ARG VERSION=latest
+
+LABEL name="kubearmor-snitch" \
+      vendor="Accuknox" \
+      version=${VERSION} \
+      release=${VERSION} \
+      summary="kubearmor-snitch container image based on redhat ubi" \
+      description="KubeArmor-Snitch, A CLI Utility to Detect node related informations for KubeArmor"
+
+ARG OPERATOR_DIR=pkg/KubeArmorOperator
+COPY --from=builder --chown=1000:1000 /KubeArmor/$OPERATOR_DIR/snitch /snitch
```

Comment:
DO we need 1000 here? If it's a root binary?

Reply:
it's a leftover i need to remove it, thanks for pointing this out.